gen-ai-sample-project/gen-ai-sample/resources/sample_table.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<database>
+    <element name="sample_table" type="collection" label="Sample table" audit="false">
+        <element name="name" label="Name" type="string"  />
+        <element name="surname" label="Surname" type="string" />
+        <element name="dashboard" label="Dashboard" type="reference" />
```

Comment:
@openai is this reference definition correct?

Reply:
@openai this is a servicenow dictionary file. Can you reevaluate if this defiinition is complete?